### DISCUSSION TITLE
Same style every time after training sd 1.5

### DISCUSSION THREAD

**Discussion by brian-tam** [TheLastBen/fast-stable-diffusion#616] [Q&A]
Hello everyone, after I trained my model using my face, the model generates the same style every time whatever my prompt typed.  Does anyone have the same problem? Even though choose oil painting, it still showing this style
![tmp8j1bcvif](https://user-images.githubusercontent.com/95268986/203931892-11d0978f-56dd-4a68-a43d-1cc695845a54.png)

**Comment by TheLastBen** [comment]
Hi, what did you name your instance images ?

**Reply by brian-tam** [reply]
Hi Ben, I name my instance to my real name plus some letters.

**Reply by TheLastBen** [reply]
don't use your real name or any existing word, use something like : "aabbcckk.jpg"

**Reply by brian-tam** [reply]
Sure! let me try to train it again. If I use my face as training set (random name as instance name), does it train the "person" class/ model in my model at the same time? Thanks Ben!

**Reply by TheLastBen** [reply]
If you use the new method, you don't have to specify the class, it will recognize it automatically, all you have to do is rename the pictures to a unique rare identifier

**Reply by brian-tam** [reply]
Thank you Ben. I just change my instance to a unique name like (tttwwwbbbhhh.jpg) and trained it overnight, but it still generates a similar style every time I run the model. But it looked more like me
I trained it with 50 pictures, 10000 steps (200 *50), turned off Enable_text_encoder_training (contains faces), turned off fp16, Train_text_encoder_for: 50.
Does it have anything to do with my training step? Is it too many steps with the quantity of the train picture set?

**Reply by TheLastBen** [reply]
>turned off Enable_text_encoder_training

if you turn off the text_encoder you won't get any results, resume the training for 1k steps but this time activate the text_encoder_training and set it to 100%

**Reply by brian-tam** [reply]
it turned on the text_encoder this time, and trained it for 1k steps and 100% for 50 pictures. 

my prompt is "portrait of tttwwwhhhbbb as a greek god, 30 years old, rugged, male, gorgeous, detailed face, amazing, hairy torso, muscular, intricate, highly detailed, digital painting, artstation, concept art, sharp focus, illustration, art by greg rutkowski and alphonse mucha"...

but still getting the similar result... 
I used 50 sampling steps and ticked high-res fix, cfg is 7, and other settings are default...
